_layouts/release_221.html

```diff
@@ -0,0 +1,289 @@
+                  <div class="col-xs-10">
+                    <a dl="javadoc">Javadoc</a>
+                    <p>
+                      API documentation.
+                    </p>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </section>
+
+          <section>
+            <h2>Source Code</h2>
+
+            <div class="row">
+              <div class="col-sm-6">
+                <div class="row">
+                  <div class="col-xs-2">
+                    <img src="/img/dl-src.png" class="img-responsive"></img>
+                  </div>
+                  <div class="col-xs-10">
+                    <a href="{{ site.git_url }}/archive/{{ page.version }}.zip">zip</a> |
+                    <a href="{{ site.git_url }}/archive/{{ page.version }}.tar.gz">tar.gz</a>
+                    <p>
+                      Source code packages for tag <a href="{{ site.git_url }}/releases/tag/{{ page.version }}">{{ page.version }}</a>.
+                    </p>
+                  </div>
+                </div>
+              </div>
+
+              <div class="col-sm-6">
+                <div class="row">
+                  <div class="col-xs-2">
+                    <img src="/img/dl-src.png" class="img-responsive"></img>
+                  </div>
+                  <div class="col-xs-10">
+                    <a href="{{ site.git_url }}/tree/{{ page.version }}">GitHub</a>
+                    <p>
+                      Browse source code on GitHub. 
+                    </p>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </section>
+
+          <section>
+            <h2>Extensions</h2>
+            <div class="row">
+              <div class="col-sm-6">
+                <div class="row">
+                  <div class="col-xs-2">
+                    <img src="/img/dl-download.png" class="img-responsive"></img>
+                  </div>
+                  <div class="col-xs-10">
+                    <a href="{{site.sf_url}}/files/GeoServer/{{page.version}}/extensions">Extensions</a>
+                    <p>
+                      GeoServer Extension downloads.
+                    </p>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="row">
+              <div class="col-sm-6">
+                <h4>Vector Formats</h4>
+                <ul>
+                  <li><a dl="app-schema-plugin">App Schema</a></li>
+                  <li><a dl="db2-plugin">DB2</a></li>
+                  <li><a dl="h2-plugin">H2</a></li>
+                  <li><a dl="mysql-plugin">MySQL</a></li>
+                  <li><a dl="oracle-plugin">Oracle</a></li>
+                  <li><a dl="feature-pregeneralized-plugin">Pregeneralized Features</a></li>
+                  <li><a dl="sqlserver-plugin">SQL Server</a></li>
+                  <li><a dl="teradata-plugin">Teradata</a></li>
```

Comment:
teradata was moved to unsupported/community module in [GEOS-10396](https://osgeo-org.atlassian.net/browse/GEOS-10396) Downgrade jt-jdbc-teradata to unsupported

```suggestion
```

Reply:
goot catch!